app/controllers/api/v1/rents_controller.rb

```diff
@@ -1,15 +1,20 @@
 module Api
   module V1
-    class RentsController < ApplicationController
+    class RentsController < ApiController
+      rescue_from Pundit::NotAuthorizedError, with: :pundit_not_authorized_error
```

Comment:
you should put this in `ApiController` instead.

Reply:
Done